func_adl_xAOD/__init__.py

```diff
@@ -1 +1,2 @@
-from .xAODlib.atlas_xaod_executor import atlas_xaod_executor # NOQA
+# from .atlas.xaod.executor import atlas_xaod_executor  # NOQA
+# from .cms.aod.cms_aod_executor import cms_aod_executor  # NOQA
```

Comment:
How is this getting marked as a "todo" so we don't forget it for future circiling back to.

Reply:
I have added it to an issue already: https://github.com/sudo-panda/func_adl_xAOD/issues/1 

If marking it as TODO will be better I will do that instead